Sources/Shared/UI/Documents/FileActionsViewController.m

```diff
@@ -329,7 +336,13 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
         }
 	}
 }
-
+-(void) alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)buttonIndex {
+    if (alertView == self.fileDeleteConfirmationBox){
```

Comment:
Set a custom `tag` on the `alertView` when you create it, and use this for comparison here, rather than comparing the instances.